pkg/apiserver/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -196,6 +196,26 @@ func admitDVs(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 		return toAdmissionResponseError(err)
 	}
 
+	client := GetClient()
+	if client != nil {
+		pvcs, err := client.CoreV1().PersistentVolumeClaims(dv.GetNamespace()).List(metav1.ListOptions{})
+		if err != nil {
+			return toAdmissionResponseError(err)
+		}
+		for _, pvc := range pvcs.Items {
+			if pvc.Name == dv.GetName() {
```

Comment:
Are we validating that pvc.name = dv.name somewhere? Just in case someone changes the code to create another name for the PVC, we should not allow this.

Reply:
Actually, isn't better to check the name of the ownerReferences in the pvc?

Reply:
I'm not sure I got you, we want to block the option of creating dv with already existing pvc name in the same namespace.

Reply:
This check is explicitly for when a pvc not owned by a dv already exists, so there won't be an owner reference.

Reply:
yeah... I was confused for a second and thought I delete this comment. you two definitely right.